app/v1/controllers/requestParsers/validators/validations/DateToValidation.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.controllers.requestParsers.validators.validations
+
+import java.time.LocalDate
+
+import v1.models.errors.MtdError
+
+import scala.util.{Failure, Success, Try}
+
+object DateToValidation {
```

Comment:
This is the same validation as the DateFromValidation, so maybe we could collapse it down to a single DateValidation and use the error we pass it to tell us which parameter we are validating